Dockerfile

```diff
@@ -1,5 +1,20 @@
-FROM alpine
+FROM golang:1.13.0-alpine3.10 AS builder
+
+WORKDIR /app
+RUN apk --no-cache add git make
```

Comment:
perhaps we should build a base image with all the tools in it

Reply:
Will do that.